src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -894,7 +894,7 @@ def test_ingress_config(self, _run_cephadm, cephadm_module: CephadmOrchestrator)
                                 'auth_type PASS\n      '
                                 'auth_pass 12345\n  '
                                 '}\n  '
-                                'unicast_src_ip 1::4\n  '
```

Comment:
I remember that we had a lot of pain with ipv6 here. would it make sense to add a test specific for IPv6 to avoid regessions?

Reply:
> I remember that we had a lot of pain with ipv6 here. would it make sense to add a test specific for IPv6 to avoid regessions?

I made the local ip (an ipv4 ip in this case) explicit instead of using the default value `1::4` to make the UT more readable. I'll check if I can add a test that covers ipv6 as well :+1: